polysh/control_commands.py

```diff
@@ -136,7 +135,7 @@ def do_send_ctrl(command):
     control_letter = chr(ord(letter.lower()) - ord('a') + 1)
     for i in selected_shells(' '.join(split[1:])):
         if i.enabled:
-            i.dispatch_write(control_letter)
+            i.dispatch_write(control_letter.encode('utf8'))
```

Comment:
`'utf-8'` is the default.  Let's not repeat it everywhere.  Also `utf8` spelling is wrong.

Reply:
Done